cli_reference/developer_cli_odo/odo-release-notes.adoc

```diff
@@ -1,74 +1,92 @@
+link:https://github.com/redhat-developer/odo/issues/5546[(odo-5546)]
 
-If you find an error or have suggestions for improving the documentation, file an issue in link:http://bugzilla.redhat.com[Bugzilla]. Choose the *{product-title}* product type and the *Documentation* component type.
+* Before this update, the `odo dev` command did not use the `.gitignore` file when synchronizing files to a container. With this update, `odo` uses the `.gitignore` file.
+link:https://github.com/redhat-developer/odo/issues/5581[(odo-5581)]
 
+* Before this update, the `odo deploy` and `odo build-images` did not work on Windows. With this update, `odo` can now build images on Windows.
+link:https://github.com/redhat-developer/odo/issues/5588[(odo-5588)]
 
+* Before this update, the `odo delete` command failed for Kubernetes components. This update fixes the issue.
+link:https://github.com/redhat-developer/odo/issues/5637[(odo-5637)]
 
+* With this update, `odo` now supports the `command` field in container components.
+link:https://github.com/redhat-developer/odo/issues/5648[(odo-5648)]
 
+* Before this update, when you stopped the `odo dev` process, resources created using `odo deploy` were removed. With this update, `odo` only deletes resources that were deployed using `odo dev`.
+link:https://github.com/redhat-developer/odo/issues/5700[(odo-5700)]
 
-////
-[id="odo-known-issues_{context}"]
-== Known issues
+* Before this update, `odo` would display volumes defined in the devfile as containers. This update fixes the issue.
+link:https://github.com/redhat-developer/odo/issues/5779[(odo-5779)]
 
-////
+* Before this update, `odo` would save the component name in the `env.yaml` file. With this update, the component name is only specified in the devfile.
+link:https://github.com/redhat-developer/odo/issues/5780[(odo-5780)]
 
-//[id="odo-technology-preview_{context}"]
-//== Technology Preview features `{odo-title}`
+
+* Before this update, after terminating the `odo dev` command, `odo` would delete the `ServiceBinding` resource but the logs would have an entry stating "Failed to delete the `ServiceBinding` resource". This update fixes the issue.
+link:https://github.com/redhat-developer/odo/issues/5797[(odo-5797)]
+
+* Before this update, if you ran the `odo dev` command on a white background terminal, the colours could look wrong or the content could be hard to read. With this update, the content is displayed correctly.
+link:https://github.com/redhat-developer/odo/issues/5843[(odo-5843)]
+
+* Before this update, the `odo logs` command could be slow to respond in certain environments. With this update, the command responds as expected.
+link:https://github.com/redhat-developer/odo/issues/5872[(odo-5872)]
+
+* Before this update, the `odo dev port forward` command could timeout if the application is not ready. This update fixes the issue.
```

Comment:
```suggestion
* Before this update, the `odo dev port forward` command might timeout if the application is not ready. This update fixes the issue.
```
Per ISG: Don't use "could" as a synonym for "might" or "can".

Reply:
I think "could" is ok for past tense, but "might" works as well so will change here